docs/STATE.md

```diff
@@ -16,17 +16,18 @@ and update this document as the code evolves.
 ## Where we are now
 
 * App written in a mixture of Java and Kotlin
-* App has mixture of unit tests (JUnit), Robolectric tests (Junit + Robolectric) and Espresso tests but coverage is far from complete
+* App has mixture of unit tests, ["Local tests" and "Instrumented tests"](https://developer.android.com/training/testing/fundamentals#instrumented-vs-local) but coverage is far from complete
```

Comment:
Android docs now have clearer language on this split so wanted to reflect that here.